internal/subshell/subshell.go

```diff
@@ -108,7 +120,7 @@ func New() SubShell {
 	case "cmd":
 		subs = &cmd.SubShell{}
 	default:
-		logging.Errorf("Unsupported shell: %s, defaulting to OS default.", name)
+		logging.Debug("Unsupported shell: %s, defaulting to OS default.", name)
```

Comment:
I wonder if it is worthwhile creating an analytics event for this one.  So, we can find out what the most prevalent shells are that we are not supporting...

Reply:
Good idea! Please file a ticket :)

Reply:
https://www.pivotaltracker.com/story/show/177629477